test/extensions/filters/network/kafka/broker/filter_unit_test.cc

```diff
@@ -248,10 +248,10 @@ TEST_F(KafkaMetricsFacadeImplUnitTest, ShouldRegisterResponse) {
   const int32_t correlation_id = 1234;
   AbstractResponseSharedPtr response = std::make_shared<MockResponse>(api_key, correlation_id);
 
-  MonotonicTime request_time_point{MonotonicTime::duration(1234000000)};
+  MonotonicTime request_time_point{std::chrono::microseconds(1234000000)};
```

Comment:
Maybe convert these to milliseconds or seconds to remove some zeros?

Reply:
converted to milliseconds, that looks like what the implementation converts the diffs in time to